index.html

```diff
@@ -0,0 +1,137 @@
+                        <span class="bar"></span>
+                        <span class="bar" id="btn2"></span>
+                    </div>
+                </div>
+            </nav>
+        </header>
+        
+        <div class="heading">
+            <p class="title">Making Differences in the life of others</p>
+            <h1>Leading the Way in Medical Excellence</h1>
+            <p class="description">
+                Empowering Lives Through Exceptional Medical Care,
+                Journey to Health and Welliness starts Here.
+            </p>
+            <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+            <p class="second-title">
+                Patience is the road to wisdom.
+            </p>
+        </div>
+    </section>
+
+    <!---Popular Services section--->
+    <section class="main-program">
+        <h1>Popular Services</h1>
+        <div class="cards-container">
+            <div class="card">
+                <img src="./images/base/program_icon_01.png" alt="">
+                <p class="card-title">Surgery</p>
+                <p class="card-info">Our comprehensive facilities include laboratory and imaging services, day care surgery, specialist consultation, medical treatment, and rehabilitative </p>
+            </div>
+            <div class="card">
+                <img src="./images/base/program_icon_02.png" alt="">
+                <p class="card-title">Obstetrics</p>
+                <p class="card-info">The Obstetrics and Gynecology Department is committed to providing the highest quality of care for the reproductive and gynecological health of women </p>
+            </div>
+            <div class="card">
+                <img src="./images/base/program_icon_03.png" alt="">
+                <p class="card-title">Cardiology</p>
+                <p class="card-info">Cardiology. Department of CardiologyThe Department of Cardiology at Tree Top Hospital diagnoses and treats wide range of conditions of the heart </p>
+            </div>
+            <div class="card">
+                <img src="./images/base/program_icon_04.png" alt="">
+                <p class="card-title">Urology</p>
+                <p class="card-info">Department of Urology focuses on medical and surgical diseases of the urinary tract, including the kidneys, ureters, bladder, urethra in both genders</p>
+            </div>
+        </div>
+        <div class="btn-container">
+            <button class="services-btn" type="button">Learn More</button>
+        </div>
+    </section>
+
+    <!--- Specialist Doctors Team--->
+    <section class="specialist-doctors">
+        <h1>Specialist Doctors</h1>
+        <section class="specialist-cards"></section>
+        <button id="moreBtn" class="show-more-btn" type="button">MORE <img src="./images/base/arrow_down.png" alt=""></button>
+    </section>
+        <!--- Prtners --->
+
+    <section class="partners">
+        <h1>Partner</h1>
+        <div class="partners-container">

```

Comment:
- [ ] [REQUIRED] Good job so far! All sections should look almost identical to the original design, outside the content differences. taht should use the exact same `colors`, for `text`, `backgrounds`, `borders`, `elements`, `shapes`, etc, from the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/63915024/6aa4f698-27db-4982-a73b-922fd24b12cf)